src/Network/Network/Models/NetworkManager/PSReachabilityAnalysisRunProperties.cs

```diff
@@ -0,0 +1,28 @@
+// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Management.Network.Models;
+using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.Commands.Network.Models.NetworkManager
+{
+    public class PSReachabilityAnalysisRunProperties
```

Comment:
Check the swagger doc for additional read-only properties here. We should have intentContent, analysisResult, and errorMessage. Also for the results, we should try to parse the json so it is readable for the customer.